cirq-core/cirq/ops/raw_types.py

```diff
@@ -342,16 +364,18 @@ def controlled(
         """Returns a controlled version of this gate. If no arguments are
         specified, defaults to a single qubit control.
 
-         num_controls: Total number of control qubits.
-         control_values: For which control qubit values to apply the sub
-             gate.  A sequence of length `num_controls` where each
-             entry is an integer (or set of integers) corresponding to the
-             qubit value (or set of possible values) where that control is
-             enabled.  When all controls are enabled, the sub gate is
-             applied.  If unspecified, control values default to 1.
-         control_qid_shape: The qid shape of the controls.  A tuple of the
-             expected dimension of each control qid.  Defaults to
-             `(2,) * num_controls`.  Specify this argument when using qudits.
+
+        Args:
+            num_controls: Total number of control qubits.
+            control_values: Which control qubit values to apply the sub
+                gate.  A sequence of length `num_controls` where each
+                entry is an integer (or set of integers) corresponding to the
+                qubit value (or set of possible values) where that control is
```

Comment:
The term "qubit value" is rather vague. IIUC, this refers to a state in the computational basis, so we should replace "qubit value" with "computational basis state" since that is clear and unambiguous.